src/components/Signup/Signup.jsx

```diff
@@ -17,36 +33,55 @@ export default function Signup() {
                     <label className='popup__form-label'>
                         Email:
                         <input
+                            name='email'
+                            onChange={handleChange}
```

Comment:
If you have `onChange` or ` onInput` in an input then you need to add attribute `value` to this input.
This is how React controls inputs.
To read more about **controlled components**: https://reactjs.org/docs/forms.html#controlled-components